segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/noop/StorageExtraConfigTest.java

```diff
@@ -43,4 +43,18 @@ public void testNoOpSwitch() {
         builder.with(Property.named("storageNoOpMode"), true);
         assertEquals(true, builder.build().isStorageNoOpMode());
     }
+
+    @Test
+    public void testSlowModeEnabled() {
+        ConfigBuilder<StorageExtraConfig> builder = StorageExtraConfig.builder();
+        builder.with(Property.named("slow.enable"), true);
+        assertEquals(true, builder.build().isSlowModeEnabled());
+    }
+
+    @Test
+    public void testSlowModeLatency() {
+        ConfigBuilder<StorageExtraConfig> builder = StorageExtraConfig.builder();
+        builder.with(Property.named("slow.latency.ms"), 500);
```

Comment:
I don't think the property name here correct.
It should be `storageextra.slow.latency.mean.ms`
also please value other than default like 42